firmware/boron-ins-fsm/test/mocks/helper.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Contains port of Linux function clock_gettime(CLOCK_MONOTONIC)
+ * To be used in millis() mock function for Windows machines
+ */
+
+#pragma once
```

Comment:
What does `#pragma once` do? 
Is it needed for these mocks to work properly?
Is it just to get around the one definition rule (as described [here](https://stackoverflow.com/questions/14517546/how-can-a-c-header-file-include-implementation)) because the implementations of these functions in in the .h file?

Reply:
#pragma once causes the files to only be included once during compilation. I wouldn't call it a workaround so much as a safeguard, in case the file is included in multiple files unintentionally, I'm guessing the mocks would still work without it, but I think including header guards, either using #pragma once or #ifndef is considered good practice

Reply:
Thanks for the explanation. I've used and seen `#ifndef` before. Nice to know that there is this one one too.